docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/jwt-authentication.md

```diff
@@ -0,0 +1,22 @@
+---
+title: JWT Authenticaiton
+---
+<!-- Edit Canonical Link -->
+<head> 
+  <link rel="canonical" href="https://ranchermanager.docs.rancher.com/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/jwt-authentication"/>
+</head>
+
+Many 3rd party integrations available for Kubernetes, such as Gitlab and Hashicorp Vault, involve giving an external process access to the Kubernetes API using a native Kubernetes Service Account token for authentication.
+
+In Rancher v2.9.0 and later, you can enable a downstream cluster to support JWT authentication of tokens created for a service account on a downstream cluster through the Rancher authentication proxy. Prior to this feature, Rancher rejects such requests because Rancher only supports Rancher-issued tokens.
+
+:::note
+JWT authentication is also known as service account token authentication.
+:::
```

Comment:
```suggestion
JWT authentication is also known as service account token authentication.
```

Is that generally true, or is it just that Rancher uses JWT for service account authentication?

Reply:
Not strictly true.
Service Account Token Auth happens to be JWT-based (and in the original github issue, we call this feature Service account token auth), but to say that JWT auth is a.k.a. Service account token auth would be incorrect.  JWTs have many other uses outside of Rancher or service account token auth.

Reply:
@crobby Thanks for confirming. I removed this.